tests/test_engine.py

```diff
@@ -15,11 +18,15 @@
 from .doubles import FakeThreadedReactor, ImmediateWorker
 
 
-def create_engine():
+def create_engine(**kwargs):
```

Comment:
This should work as you say currently, but in separate tox testenvs. It's slower than it could be... but it keeps the current testing code substantially similar (and simpler?).

I reverted adding the psycopg2cffi to the default tox testenvs. So the current behavior is testing sqlite only locally, but manually specifying the extra envs in the .travis.yml so the full suite runs there.